frontend/src/uxbox/main/data/workspace.cljs

```diff
@@ -464,10 +465,11 @@
             uchange  {:type :del-obj
                       :id id}]
 
-        (rx/of (dwc/commit-changes [rchange] [uchange] {:commit-local? true})
-               (dws/select-shapes (d/ordered-set id))
-               (when (= :text (:type attrs))
-                 (start-edition-mode id)))))))
+        (rx/concat
+         (rx/of (dwc/commit-changes [rchange] [uchange] {:commit-local? true})
+                (dws/select-shapes (d/ordered-set id)))
+         (->> (rx/of (start-edition-mode id))
```

Comment:
It's no longer necessary to checkt that the shape type is text?

Reply:
you are right, i'll fix it

Reply:
fixed